index.php

```diff
@@ -45,7 +48,9 @@
             <label>Indeks: <br /><input type="number" name="index" /></label>
             <label>Funkcja: <br /><input type="text" name="function" /></label>
             <label>Przewodniczący: <br /><input type="text" name="leader" /></label>
-
+            <label>Nazwa Koła: <br /><input type="text" name="club_name" placeholder="np.: Akademickiego koła aplikacji internetowych"/></label>
```

Comment:
W nawiasie warto dodać, że w dopełniaczu, bo z placeholdera nie wynika to tak jasno bo zazwyczaj każdy go pomija, a tutaj trzeba zwrócić szczególną uwagę na to aby tak właśnie tę nazwę wpisać.